test/index.js

```diff
@@ -15,167 +15,138 @@ const internals = {};
 
 // Test shortcuts
 
-const lab = exports.lab = Lab.script();
-const describe = lab.describe;
-const it = lab.it;
-const expect = Code.expect;
+const { describe, it } = exports.lab = Lab.script();
```

Comment:
You can also get the `expect` in this destructuring.

Reply:
Wouldn't it then get `expect` from lab instead of getting it from `Code`?